packages/client/src/v2-events/features/events/registered-fields/index.ts

```diff
@@ -27,3 +57,51 @@ export * from './Paragraph'
 export * from './Name'
 export * from './SelectDateRangeField'
 export * from './Http'
+
+export function getFieldByType<T extends FieldType>(
+  type: T
+): RegisteredFieldModule<FieldTypeToFieldConfig<T>> | undefined {
+  // eslint-disable-next-line @typescript-eslint/switch-exhaustiveness-check
+  switch (type) {
+    case FieldType.ADDRESS:
+      return Address
+    case FieldType.BULLET_LIST:
+      return BulletList
+    case FieldType.BUTTON:
+      return Button
+    case FieldType.CHECKBOX:
+      return Checkbox
+    case FieldType.DATE:
+      return DateField as any
```

Comment:
I need to look at these anys with a fresh pair of eyes. Just could not get these to work today